### PR TITLE
Added GoToCoordWhileAiming task

### PR DESCRIPTION
**Pull request by butich**

### CODE REVIEW

File: source/scripting/Tasks.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using System;
+using System.Collections.Specialized;
```
Comment (crosire) [inline source/scripting/Tasks.cs]: What is this used for?

File: source/scripting/Tasks.cs [hunk 2]
```diff
@@ -185,6 +186,22 @@ public void GoTo(Vector3 position, bool ignorePaths = false, int timeout = -1)
 			}
 		}
 
+	    public void GoToCoordWhileAiming(Vector3 position, Vector3 aimCoord, bool shooting, FiringPattern pattern)
+	    {
+	        if (shooting)
+	        {
+	            Function.Call(Hash.TASK_GO_TO_COORD_WHILE_AIMING_AT_COORD, _ped.Handle, position.X, position.Y,
+	                position.Z, aimCoord.X, aimCoord.Y, aimCoord.Z, 2f, 1, 0x3F000000, 0x40800000, 1, 0, 0,
+	                (uint) pattern);
```
Comment (crosire) [inline source/scripting/Tasks.cs]: `Function.Call` accepts enums now, so the cast to `uint` is not necessary.

**Comment by butich** [comment]
Oh, thanks

**Comment by crosire** [comment]
No need to close the pull request. Just push a follow-up commit to the same branch and it is updated automatically :smiley:.

**Comment by butich** [comment]
I'm new at github;) Sorry

File: source/scripting/Tasks.cs (hunk 2 above)
Comment (zorg93) [inline source/scripting/Tasks.cs]: This is a minor one but its a pet hate of mine. that 0x3F000000 should be 0.5f, and 0x40800000 should be 4.0f. writing them as floats makes it more understandable for people when reading the code

File: source/scripting/Tasks.cs [hunk 3]
```diff
@@ -189,11 +189,11 @@ public void GoToCoordWhileAiming(Vector3 position, Vector3 aimCoord, bool shooti
 	    {
             if (shooting)
             {
-                Function.Call(Hash.TASK_GO_TO_COORD_WHILE_AIMING_AT_COORD, _ped.Handle, position.X, position.Y, position.Z, aimCoord.X, aimCoord.Y, aimCoord.Z, 2f, 1, 0x3F000000, 0x40800000, 1, 0, 0, (uint)pattern);
+                Function.Call(Hash.TASK_GO_TO_COORD_WHILE_AIMING_AT_COORD, _ped.Handle, position.X, position.Y, position.Z, aimCoord.X, aimCoord.Y, aimCoord.Z, 2f, 1, 0.5f, 4, 1, 0, 0, pattern);
```
Comment (zorg93) [inline source/scripting/Tasks.cs]: sorry to be a pain but its 4f not 4. same for below.

**Pull request closed without merging**